src/main/java/edu/kis/powp/jobs2d/command/manager/CommandParser.java

```diff
@@ -0,0 +1,34 @@
+package edu.kis.powp.jobs2d.command.manager;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommandParser {
```

Comment:
CommandParser should support different formats (strategy or builder pattern). 
Let's start from introducing an interface.